### PR TITLE
[Translation] [russian] Translates welcome page into Russian

### PR DESCRIPTION
**Pull request by koleaby4**
translation of the welcome page into Russian

### CODE REVIEW

**Comment by ardan-bkennedy** [comment]
I need your email please. There is someone who will help you and I want to give you access to this repo.

**Comment by koleaby4** [comment]
negru.n@gmail.com

**Comment by koleaby4** [comment]
> I need your email please. There is someone who will help you and I want to give you access to this repo.



> I need your email please. There is someone who will help you and I want to give you access to this repo.

negru.n@gmail.com

File: _content/tour/rus/welcome.article [hunk 1]
```diff
@@ -1,211 +1,198 @@
...
-it has a unique and simple nature that make Go programs different in character from
-programs written in other languages. It balances the capabilities of a low-level
-systems language with some high-level features you see in modern languages today.
-This creates a programming environment that allows you to be incredibly productive,
-performant and fully in control; in Go, you can write less code and do so much more.
+Go - это язык программирования с открытым исходным кодом, который облегчает создание простого,
+надежного и эффективного программного обеспечения. Несмотря на то, что он заимствует идеи из существующих языков,
+у него есть уникальная и простая природа, которая отличает программы на Go от программ, написанных на других языках.
+Он совмещает возможности низкоуровневого системного языка с некоторыми высокоуровневыми функциями,
+которые вы видите в современных языках. Это создает программную среду, которая позволяет вам быть невероятно продуктивным, производительным и полностью контролируемым; на Go вы можете писать меньше кода и делать намного больше.
```
Comment (rumyantseva) [inline _content/tour/rus/welcome.article:173]:
```suggestion
Go — язык программирования с открытым исходным кодом, который позволяет легко создавать
простое, надежное и эффективное программное обеспечение. Хотя он заимствует идеи
из существующих языков, он имеет уникальную и простую природу, которая делает
программы Go отличными от программ, написанных на других языках. Он балансирует
возможности низкоуровневого системного языка с некоторыми высокоуровневыми
возможностями, которые вы видите в современных языках. Это создает программную
среду, которая позволяет вам быть невероятно продуктивным, производительным и
полностью контролировать; в Go вы можете писать меньше кода и делать гораздо больше.
```

I translated this part in a slightly different way, but please feel free to use your translation if you like it better.

File: _content/tour/rus/welcome.article [hunk 2]
```diff
@@ -1,211 +1,198 @@
...
 
-** Important Reading
+** Важное чтение
 
-Please check out this page of [[https://github.com/ardanlabs/gotraining/blob/master/reading/README.md][important reading]].
-You will find articles and videos around mechanical sympathy, data-oriented design,
-Go runtime and optimizations and articles about the history of computing.
+Пожалуйста, ознакомьтесь с этой страницей [[https://github.com/ardanlabs/gotraining/blob/master/reading/README.md][важного чтения]].
+Вы найдете статьи и видеоролики о механической симпатии, проектировании, ориентированном на данные,
+среда выполнения Go и оптимизации, а также статьи об истории вычислительной техники.
```
Comment (rumyantseva) [inline _content/tour/rus/welcome.article:189]:
```suggestion
среде выполнения Go и оптимизации, а также статьи об истории вычислительной техники.
```

File: _content/tour/rus/welcome.article [hunk 3]
```diff
@@ -1,211 +1,198 @@
...
 
-** Joining the Go Slack Community
+** Присоединяйтесь к Go Slack Community
```
Comment (rumyantseva) [inline _content/tour/rus/welcome.article:191]:
```suggestion
** Присоединяйтесь к Go-сообществу в Slack
```

File: _content/tour/rus/welcome.article [hunk 4]
```diff
@@ -1,211 +1,198 @@
...
 
-We use a slack channel to share links, code, and examples during the training.
-This is free. This is also the same slack community you will use after training to
-ask for help and interact with may Go experts around the world in the community.
+Мы используем канал Slack для обмена ссылками, кодом и примерами во время обучения.
+Это бесплатно. Это тот же Slack-комьюнити, которое вы будете использовать после обучения, чтобы
+спросить совета и взаимодействовать с экспертами по Go по всему миру в сообществе.
```
Comment (rumyantseva) [inline _content/tour/rus/welcome.article:195]:
```suggestion
Мы используем канал в Slack для обмена ссылками, кодом и примерами во время
обучения. Это бесплатно. После обучения вы будете использовать этот же канал Slack,
чтобы обращаться за помощью и взаимодействовать с экспертами Go-сообщества со всего света.
```

File: _content/tour/rus/welcome.article [hunk 5]
```diff
@@ -1,211 +1,198 @@
...
 
-- Using the following link, fill out your name and email address: [[https://invite.slack.gobridge.org/][https://invite.slack.gobridge.org]]
-- Check your email, and follow the link to the slack application.
+Используя следующую ссылку, заполните ваше имя и адрес электронной почты: [[https://invite.slack.gobridge.org/][https://invite.slack.gobridge.org]]
```
Comment (rumyantseva) [inline _content/tour/rus/welcome.article:197]:
```suggestion
- Пройдите по ссылке, введите свое имя и адрес электронной почты: [[https://invite.slack.gobridge.org/][https://invite.slack.gobridge.org]]
```

**Review by rumyantseva — COMMENTED** [review]
It's always hard to start something new, so great job creating the translation for the first chapter of the Go Tour!

I added a few suggestions for these situations:
- the initial formatting was different;
- there are some typos / wrong grammatical cases;
- I had quite a different phrasing in my version.

Overall, I don't have critical comments, and if something can be improved in the future, anyone can suggest improvements through further PRs.

**Comment by koleaby4** [comment]
> It's always hard to start something new, so great job creating the translation for the first chapter of the Go Tour!
> 
> I added a few suggestions for these situations:
> 
> * the initial formatting was different;
> * there are some typos / wrong grammatical cases;
> * I had quite a different phrasing in my version.
> 
> Overall, I don't have critical comments, and if something can be improved in the future, anyone can suggest improvements through further PRs.

Thanks for the comments and suggestions for improvements, @rumyantseva - very useful indeed.
Looking forward to working with you on this translation 👍

**Review by rumyantseva — APPROVED** [review]
LGTM

**Pull request merged**